api/envoy/data/tap/v2alpha/http.proto

```diff
@@ -10,8 +10,14 @@ import "envoy/api/v2/core/base.proto";
 // A fully buffered HTTP trace message.
 message HttpBufferedTrace {
   // Request headers.
-  repeated api.v2.core.HeaderValue request_headers = 2;
+  repeated api.v2.core.HeaderValue request_headers = 1;
+
+  // Request trailers.
+  repeated api.v2.core.HeaderValue request_trailers = 2;
 
   // Response headers.
   repeated api.v2.core.HeaderValue response_headers = 3;
+
+  // Response trailers.
```

Comment:
Do we want a request/response grouping in here?

Reply:
Do you mean in separate sub-messages?

Reply:
Yep

Reply:
Sure will do.